.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @armory/ico
```

Comment:
This will add `armory/ico` to every PR in the project. If we want to make sure someone is looking at specific parts of the codebase then we can use a more specific glob.

Reply:
Does it do something besides documenting? like an alias somewhere?

Reply:
Documenting what? This file is basically saying "for any file inside a PR (`*`), add `@armory/ico` to the pull request as an approver"

Reply:
Got it. Thanks